examples/speech_to_text/simultaneous_translation/scripts/stream_laal.py

```diff
@@ -100,6 +100,7 @@ def __call__(self, prediction: str, reference_sentences: List[str]) -> List[str]
         """
         tmp_pred = tempfile.NamedTemporaryFile(mode="w", delete=False)
         tmp_ref = tempfile.NamedTemporaryFile(mode="w", delete=False)
+        tmp_dir = tempfile.mkdtemp()  # temporary directory where mwerSegmenter writes the segments. It works in paralel
```

Comment:
```suggestion
        # create a  temporary directory where mwerSegmenter writes the segments
        # so that multiple parallel runs of stream_laal do not conflict
        tmp_dir = tempfile.mkdtemp()
```